lib/Dialect/Comb/CombFolds.cpp

```diff
@@ -1813,11 +1813,16 @@ static bool foldMuxChain(MuxOp rootMux, bool isFalseSide,
   if (valuesFound.size() < 3)
     return false;
 
+  // If the array is greater that 9 bits, it will take over 512 elements and
+  // it is too large for a regular expression.
+  auto indexWidth = indexValue.getType().cast<IntegerType>().getWidth();
+  if (9 <= indexWidth)
```

Comment:
Please change this to `(indexWidth >= 9)` for consistency with the rest of the codebase (e.g. the 'if' right above).  Thanks!